daemons/ipa-kdb/ipa_kdb_kdcpolicy.c

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+ */
+
+#include <errno.h>
+#include <syslog.h>
+#include <krb5/kdcpolicy_plugin.h>
+
+#include "ipa_krb5.h"
+#include "ipa_kdb.h"
+
+struct krb5_kdcpolicy_moddata_st {
+    /* Storage struct for kdcpolicy callback state */
+    ;
```

Comment:
why an empty struct ?
is it actually required to have a pointer at all ?

Reply:
Nope.  We could delete it entirely if we don't need any state here.  Historically, it's only in the commit at all because it made sense to expose as much in the skeleton commit as I could.

Reply:
Nope.  We could delete it entirely if we don't need any state here.  Historically, it's only in the commit at all because it made sense to expose as much in the skeleton commit as I could.

Reply:
Technically this does mean that `ipa_kdcpolicy_init` and `ipa_kdcpolicy_fini` aren't needed either - we can just make them NULLs in the vtable.